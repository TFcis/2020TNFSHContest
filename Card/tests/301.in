10
8 6 7 10 4 3 5 9 2 1
1 9 7 3 10 8 5 4 2 6
26
