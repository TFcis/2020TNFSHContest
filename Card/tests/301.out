8 6 4 5 3 10 1 7 2 9
